schema/json/vnd.syft+json

```diff
@@ -0,0 +1,85 @@
+(registered 2021-11-05, last updated 2021-11-05)
+
+Media type name: application
+
+Media subtype name: vnd.syft+json
+
+Required parameters: N/A
+
+Optional parameters:
+
+   version
+
+      The version parameter refers to the Syft specification
+      version in use.
+
+      version = 1*DIGIT "." 1*DIGIT "." 1*DIGIT
+
+Encoding considerations: binary
+
+   This media type has all of the same encoding considerations of
+   application/json as described in [RFC8259].
+
+Security considerations: This media type has all of the same security
+   considerations of application/json as described in [RFC8259].
+
+   Additional care should be taken when processing additionalProperties,
+   digest, files, fileClassifications <>..... <----------------------------TODO Let's get this complete
```

Comment:
Question: Is it acceptable to convey the meaning of this paragraph without referring to specific field names/types from the schema? In my mind, the versioning of the schema governs what's currently accessible, and I'm thinking it'd be nice not to have to remember to update this central media type definition as the schema evolves, potentially removing/changing these referenced names each time.

What do others think?

Reply:
Resolved to be less constrained to our schema. We just call out the JSON considerations from RFC8259